packages/help-center/src/components/help-center-header.tsx

```diff
@@ -46,22 +46,27 @@ const SupportModeTitle = (): ReactElement => {
 	}
 };
 const HelpCenterHeader: React.FC< Header > = ( {
-	isMinimized,
+	isMinimized = false,
 	onMinimize,
 	onMaximize,
 	onDismiss,
 } ) => {
+	const { push } = useHistory();
+	const { unreadCount, chatStatus, closeChat } = useHCWindowCommunicator( isMinimized );
 	const classNames = classnames( 'help-center__container-header' );
 	const { __ } = useI18n();
-	const [ , setChatWindowStatus ] = useState< WindowState >( 'closed' );
-	const [ unreadCount, setUnreadCount ] = useState< number >( 0 );
 	const formattedUnreadCount = unreadCount > 9 ? '9+' : unreadCount;
 
-	function requestMaximize() {
-		onMaximize?.();
-	}
+	useEffect( () => {
+		if ( chatStatus === 'ended' ) {
+			push( '/' );
+		}
+	}, [ chatStatus, push ] );
```

Comment:
We shouldn't close the chat when the HE ends it. In case the user wants to say something more.

Reply:
I think that is the current flow with HappyChat. HE's have the ability to `/end` the chat and close it out. After the chat is closed the user is brought to the HelpCenter home and is welcome to start a new chat if they have another question.

In general, I think having clear closes is important. The one improvement I think would be needed is a message saying "Thanks for chatting, we have emailed you the transcripts from this conversation"

Reply:
Well, Igor [explicitly told us](https://github.com/Automattic/wp-calypso/pull/66407#:~:text=new%20sessions%20instead.-,High%20priority%3A%20Another%20change%20of%20behaviour%3A%20when%20the%20chat%20is%20ended%2C%20the%20Help%20Center%20immediately%20reverts%20to%20the%20main%20view.,-That%27s%20a%20very) that the user shouldn't be removed from the conversation. Because occasionally, they want to read back and apply the instructions.

Reply:
>Because occasionally, they want to read back and apply the instructions.

I think that would be the intent of letting them know the conversation was emailed to them.

>Well, Igor explicitly told us that the user shouldn't be removed from the conversation.

I tested this in our current setup and it seemed a little disconnected. When an HE closes the chat the window is left unchanged from the users perspective. Giving them the impression that the chat is still open and the session has not changed. If the user sends another message like "Bye" or "Thanks again!" it creates a new chat and if the previous HE is gone then the chat is opened with a new HE. Here is the transcript from my testing - 36814543-hc

I think it would be helpful to have some sort of indicator that the session was closed and point the user to the transcript that was emailed to them. We do not inform them at all that the conversation was sent to them. As an HE, it was common to get a chat with a user just saying "Thanks" because they think they are still in the chat, or trying to pickup exactly where they left off.

Maybe sending them back to the HelpCenter home with a message isn't the right answer, but I think at the very least we should make the user aware that the session was closed and they could start another if they like by submitting a new message. @klimeryk

Reply:
100% agree with this ☝🏼 But let's not make it a blocker, we're way late. Let's remove the redirect because it's harsher than the silence and create an issue for the "end of chat" indication. I totally think it's worth it to show some indication like "The agent ended this chat. You can start a new session anytime!".